client/k8s/k8s.go

```diff
@@ -44,7 +43,6 @@ func newRawClientConfig(config Config) *rest.Config {
 		Username:        config.Username,
 		Password:        config.Password,
 		BearerToken:     config.BearerToken,
-		Insecure:        config.Insecure,
```

Comment:
I'm not using the insecure field. I think we should remove this client and use the one in `operatorkit`.  Either in this PR or soon.

https://github.com/giantswarm/operatorkit/tree/master/client/k8s